RubeesEat/Pages/BillDetails.cshtml

```diff
@@ -2,33 +2,42 @@
 @using System.Globalization
 @inject IBillRepository billRepository
 
-@section Styles {
-<link rel="stylesheet" href="/css/BillDetails.css" />
-}
 @{
     var billId = Guid.Parse(RouteData.Values["guid"].ToString());
```

Comment:
Preexisting issue: why use ToString on the RouteData-Value? shouldn't this already be a string. -> Ticket please.

Reply:
`RouteData.Values["guid"]` is an object and not a string